gr-azure-software-radio/python/blob_sink.py

```diff
@@ -21,15 +22,15 @@
 
 
 class BlobSink(gr.sync_block):
-    """ Write samples out to an Azure Blob.
+    """ Write samples out to an Azure Blob.  Use a Head block to limit the number of samples stored in the file.
 
     This block has multiple ways to authenticate to the Azure blob backend. Users can directly
     specify either a connection string, a URL with an embedded SAS token, or use credentials
     supported by the DefaultAzureCredential class, such as environment variables, a
     managed identity, the az login command, etc.
 
-    This block currently only supports complex64 inputs and only supports block blobs. Page blobs
-    and append blobs are not supported.
```

Comment:
we should keep the "only supports block blobs. Page blobs and append blobs are not supported" statement.

Reply:
Done, i also added the note to source block